### PR TITLE
Blog post for 0.18.3: SoundCloud Extractor hotfix

### PR DESCRIPTION
**Pull request by Stypox**
As requested in TeamNewPipe/NewPipe#3072 I wrote a small article about the new release :-)
This is the first time I open a PR in the website, so I hope I did all the needed things in order to publish a new blog post. ;-)

### CODE REVIEW

File: _posts/2020-02-10-newpipe-0.18.3-released.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.3 released: Hotfix for the SoundCloud Extractor"
+short: "NewPipe 0.18.3 released"
+date: 2020-02-10 00:00:00 +0100
```
Comment (TheAssassin) [inline _posts/2020-02-10-newpipe-0.18.3-released.md]: Date doesn't quite fit, should probably be `2020-02-11`.

File: _posts/2020-02-10-newpipe-0.18.3-released.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+categories: [release, pinned]
+author: Stypox
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A change on SoundCloud side made it necessary to make some edits to the SoundCloud Extractor, so a new update for NewPipe is here!
+
+## Hotfix
```
Comment (TheAssassin) [inline _posts/2020-02-10-newpipe-0.18.3-released.md]: I'm not a fan of including any kind of headlines in excerpts. One shouldn't include half of the article and split off the rest. IMO the text above is sufficient to inform people what the post is about, anyone who wants more details can open the full article.

File: _posts/2020-02-10-newpipe-0.18.3-released.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Since this is a hotfix release, it does not contain any new feature or bugfix, except for the SoundCloud Extractor fixup. If you see any Guru Mediation error in the app, please update to 0.18.3 before you create an issue (you can verify if you actually have `0.18.3+` installed by tapping `About` in the app drawer menu).
+
+<!-- more -->
+
+SoundCloud changed the API key (called `client_id`), as it uses to do rouughly once every 1-2 months. The app's hardcoded client id was updated by [@TobiGr](https://github.com/TobiGr). Further investigation is required to understand why the automatic `client_id` extraction failed, after the hardcoded id became invalid.
```
Comment (TheAssassin) [inline _posts/2020-02-10-newpipe-0.18.3-released.md]: `rooughly` should be `roughly`. `ID` should be capitalized. There should be no comma after `failed`.

File: _posts/2020-02-10-newpipe-0.18.3-released.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+excerpt_separator: <!-- more -->
+---
+
+A change on SoundCloud side made it necessary to make some edits to the SoundCloud Extractor, so a new update for NewPipe is here!
+
+<!-- more -->
+
+Since this is a hotfix release, it does not contain any new feature or bugfix, except for the SoundCloud Extractor fixup. If you see any Guru Mediation error in the app, please update to 0.18.3 before you create an issue (you can verify if you actually have `0.18.3+` installed by tapping `About` in the app drawer menu).
+
+<!-- more -->
```
Comment (TheAssassin) [inline _posts/2020-02-10-newpipe-0.18.3-released.md]: Now we have the separator twice. That should be fixed. By the way, I didn't mind the headline itself, only the placement of the separator.

File: _posts/2020-02-10-newpipe-0.18.3-released.md (hunk 2 above)
Comment (Stypox) [reply _posts/2020-02-10-newpipe-0.18.3-released.md]: I agree with you, the intermediary title is quite useless, but what I did is just copying the layout of the 0.18.2 article ;-)
Is it ok now?

Comment (TheAssassin) [reply _posts/2020-02-10-newpipe-0.18.3-released.md]: We've had a similar discussion there. Yeah, it's okay now. There's one remaining issue I've found, when it's fixed we can merge and deploy the new page.

File: _posts/2020-02-10-newpipe-0.18.3-released.md (hunk 4 above)
Comment (Stypox) [reply _posts/2020-02-10-newpipe-0.18.3-released.md]: Fixed, is ot ok?

**Pull request merged**